guides/image-sources/provisioning-installer-bootdisk-fullhost.plantuml

```diff
@@ -0,0 +1,61 @@
+@startuml
+
+!include foreman.pstyle
+
+title Installer-based provisioning with full-host bootdisk
+
+actor       User
+participant "Provisioned\nHost" as Host
+participant "Foreman" as Foreman
+participant "Foreman\nProxy" as Proxy
+participant "Pulp\n(Katello)" as Pulp
+'participant TFTP
+participant DHCP
+participant DNS
+!if ($puppet)
+participant "Puppet\nserver" as Puppet
+!endif
+
+note over Host : powered off
+
+== Create host in Foreman ==
+
+User -> Foreman : clicks **Create Host**
+Foreman -> Proxy : requests free IP address
+Proxy -> DHCP : reads DHCP cofiguration
+note over Foreman : pings IP address
+User -> Foreman : submits the **Create host** form
+group Templates [*PXE* installer boot]
+    Foreman -> Proxy : renders bootloader configuration files
+end
+Proxy -> Proxy : deploys bootloader configuration files
+Foreman -> Proxy : commands to download OS installer kernel\nand initial RAM disk
+Proxy -> Pulp : downloads OS installer kernel\nand initial RAM disk
```

Comment:
Isn't this happening  **only** if Katello is installed and only for RHEL operating systems? Like getting the data from the Kickstart repository?

Otherwise, they are downloaded from the media URL, which could be somewhere on the internet.